CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 6.11.2 - 2022/04/28
```

Comment:
Move to the next minor version for enhancements (i.e. 6.12.0).

Also note that I have a PR in flight at the moment that could go together.

Reply:
Moved version and removed date to wait for your change :)